packages/hackathon/src/scenes/HomeScene/HomeScene.tsx

```diff
@@ -42,9 +43,18 @@ export const HomeScene: FC<HomeSceneProps> = ({ hacker }) => {
   return (
     <>
       <SpaceVertical gap="u5">
-        <Heading as="h2" fontSize="xxxlarge" fontWeight="medium">
-          Agenda
-        </Heading>
+        <Span>
```

Comment:
Nested in span so the paragraph is not too far from Heading in SpaceVertical.

Reply:
For v.next with artifact storage we'll schedule some design review too

Reply:
Totally. I'm pretty sure I'm not following design best practices here, only aiming for what looks good. Will appreciate their input.